src/Components/Pickers/Internal/Picker.base.styles.ts

```diff
@@ -8,22 +8,34 @@ const classNames = {
 export const getStyles = (_props: IPickerBaseStyleProps): IPickerBaseStyles => {
     return {
         root: [classNames.root],
-        button: [
-            classNames.button,
-            {
-                border: '1px solid var(--cb-color-input-border)',
-                borderRadius: '50%',
-                cursor: 'pointer',
-                height: 28,
-                width: 28
-            }
-        ],
+        button: [],
```

Comment:
you can remove this style entirely now I think? Or leave it but keep the className.button part in there.

Reply:
nope, the IPickerBaseStyles return needs to have the `button` array or there's complier errors